plugins/chrome-ux-report/README.md

```diff
@@ -0,0 +1,96 @@
+## What is Chrome UX Report?
+
+The Chrome User Experience Report is powered by real user measurement of key user experience metrics across the public web, aggregated from users who have opted-in to syncing their browsing history, have not set up a Sync passphrase, and have usage statistic reporting enabled. If you are curious of what is the metrics, you can go to [UX Metrics](https://developers.google.com/web/tools/chrome-user-experience-report).
+
+## Configuring the Plugin
+
+You can add this plugin to the project follow these steps.
+
+1. Run:
+
+```sh
+$ yarn add @backstage/plugin-chrome-ux-report @backstage/plugin-chrome-ux-report-backend
+```
+
+2. Add the plugin backend:
+
+In a new file named `chromeuxreport.ts` under `backend/src/plugins`:
+
+```ts
+import { createRouter } from '@backstage/plugin-chrome-ux-report-backend';
+import { Router } from 'express';
+import { PluginEnvironment } from '../types';
+
+export default async function createPlugin({
+  logger,
+  config,
+  database,
+}: PluginEnvironment): Promise<Router> {
+  return await createRouter({ logger, config, database });
+}
+```
+
+And then add to `packages/backend/src/index.ts`:
+
+```ts
+// ...
+import chromeuxreport from './plugins/chromeuxreport';
+// ...
+async function main() {
+  // ...
+  const chromeuxreportEnv = useHotMemoize(module, () =>
+    createEnv('chromeuxreport'),
+  );
+  // ...
+
+  const apiRouter = Router();
+  // ...
+  apiRouter.use('/chromeuxreport', await chromeuxreport(chromeuxreportEnv));
+  // ...
+}
+```
+
+3. Add the plugin frontend to `packages/app/src/plugin.ts`:
```

Comment:
this is no longer necessary if the plugin uses the `plugin.provide` composability stuff!